tab_sponsorship.md

```diff
@@ -28,7 +28,7 @@ Small company logo on the sponsors page of the printed and ebook versions
 ### God Mode Sponsor (USD 4,000 / 5 Available)
```

Comment:
Note. I'd make this content a list, and the same goes for the rest of the packages, all lists and ending in full-stops. It's not being rendered as list as it is now written, therefore it's difficult to read.

Reply:
lists they are sir!